.bundle/config

```diff
@@ -0,0 +1,3 @@
+---
+BUNDLE_WITHOUT: omnibus_package
+BUNDLE_FROZEN: '1'
```

Comment:
@chefsalim --deployment does a bunch of other stuff (and in fact BUNDLE_FROZEN is one of the flags it sets). Run `bundle install --deployment` on your machine and then look in `.bundle/config` and you will see:

```
BUNDLE_FROZEN: '1'
BUNDLE_PATH: vendor/bundle
BUNDLE_DISABLE_SHARED_GEMS: '1'
```

This flag is the equivalent of `bundle install --frozen`, which is all we want to do.